.github/renovate.json

```diff
@@ -7,12 +7,22 @@
   "dependencyDashboard": true,
   "dependencyDashboardApproval": true,
   "ignorePaths": ["apps/engine/**"],
+  "npm": {"minimumReleaseAge": "3 days"},
   "postUpdateOptions": ["yarnDedupeFewer"],
   "rangeStrategy": "bump",
   "rebaseWhen": "conflicted",
   "semanticCommits": "disabled",
   "updateInternalDeps": "true",
 
+  "lockFileMaintenance": {
+    "enabled": true,
```

Comment:
We have automerge enabled in line `5` as this requires an approval anyway, so these two lines are the only real change in this PR.